sharktank/sharktank/layers/testing.py

```diff
@@ -43,3 +43,82 @@ def make_llama_attention_block_theta(
             ),
         }
     )
+
+
+def make_mmdit_block_theta(dtype: torch.dtype | None = None) -> Theta:
+    return Theta(
+        {
+            "img_attn.norm.key_norm.weight": DefaultPrimitiveTensor(  #
+                data=make_rand_torch((128,), dtype=dtype)
```

Comment:
I would move the numbers out into variables.
If not too difficult to construct them from the minimal amount of independent values. I would assume there is some dependence between all these values.

Reply:
Yeah, something like [this](https://github.com/nod-ai/shark-ai/blob/main/sharktank/sharktank/models/llama/testing.py). Also, should this+T5+CLIP dummy theta weights sit in `tests/layers/flux_testing.py` ?

Reply:
Done with variable refactor. I agree the new file location is a better idea, but it isn't directly related to the patch, so I'll send a follow up that changes this or approve one if it comes to me.